ambari-server/src/main/java/org/apache/ambari/server/api/services/parsers/JsonRequestBodyParser.java

```diff
@@ -170,4 +177,36 @@ private void processNode(JsonNode node, String path, NamedPropertySet propertySe
   private String ensureArrayFormat(String s) {
     return s.startsWith("[") ? s : '[' + s + ']';
   }
+
+  static class AmbariEscapes extends CharacterEscapes {
+
+    private final int[] escapeCodesForAscii;
+
+    public AmbariEscapes() {
+      this.escapeCodesForAscii = standardAsciiEscapesForJSON();
+      escapeCodesForAscii['/'] = CharacterEscapes.ESCAPE_STANDARD;
+    }
+
+    @Override
+    public int[] getEscapeCodesForAscii() {
+      return escapeCodesForAscii;
+    }
+
+    @Override
+    public SerializableString getEscapeSequence(int ch) {
+      return null;
+    }
+  }
+
+  /**
+   * Converts string to escaped and removes first and last symbols (double brackets)
+   * @param mapper mapper
+   * @param value string to be escaped
+   * @return escaped string
+   * @throws IOException
+   */
+  private String escapeValue(ObjectMapper mapper, String value) throws IOException {
```

Comment:
The placement of this method seems to be confusing. Can it be moved above the AmbariEscapes class?